.gitignore

```diff
@@ -2,3 +2,5 @@
 Gemfile.lock
 .bundle
 vendor
+.idea
+lib/com
```

Comment:
we shouldn't have vendor files in `lib/com`, this can be removed. `.idea` is fine